### DISCUSSION TITLE
Train DBNetpp with ConvNeXt backbone

### DISCUSSION THREAD

**Discussion by FranziSchmitt** [open-mmlab/mmocr#1675] [Q&A]
Hi, 
I've recently upgraded mmocr from version 0.6.2 to 1.x. 
In the previous version, I was able to train DBNetpp using a convnext backbone. With mmocr 1.x, I'm getting compatibility errors with mmcls and mmcv versions. I've tried a couple of variations and versions of mmcls, mmcv, and mmcv-full but all of them are resulting in some kind of incompatibility message. 

Training with the resnet backbone is not an option for me.

How can I get training of dbnetpp running with the convnext backbone?
Thanks!

**Comment by gaotongxiao** [comment]
MMOCR 1.0 is a part of OpenMMLab 2.0 - and all its OpenMMLab-series dependencies need to be upgraded. A suggested practice is to create a new virtual environment and install everything following the [Installation Guide](https://mmocr.readthedocs.io/en/dev-1.x/get_started/install.html). Similarly, you will have to switch mmclassicifaion to 1.x branch for compatibility.

For more information, you can read the [migration guide](https://mmocr.readthedocs.io/en/dev-1.x/migration/overview.html).
